plugin/src/main/scala/sbtcompendium/CompendiumPlugin.scala

```diff
@@ -22,29 +22,34 @@ import hammock.asynchttpclient.AsyncHttpClientInterpreter
 import higherkindness.compendium.models._
 import higherkindness.compendium.models.config.HttpConfig
 import sbt._
-import sbtcompendium.client.{CompendiumClient, CompendiumClientConfig}
 
 object CompendiumPlugin extends AutoPlugin {
 
   override def trigger: PluginTrigger = allRequirements
 
   object autoImport {
 
-    lazy val compendiumGenClients: TaskKey[Seq[File]] = taskKey[Seq[File]]("Generate all the clients for each protocol")
-
-    lazy val compendiumServerHost: SettingKey[String] = settingKey[String]("Url of the compendium server")
-    lazy val compendiumServerPort: SettingKey[Int]    = settingKey[Int]("Port of the compendium server")
+    import sbtcompendium.client.{CompendiumClient, CompendiumClientConfig}
+    lazy val compendiumGenClients: TaskKey[Seq[File]]   = taskKey[Seq[File]]("Generate all the clients for each protocol")
+    lazy val compendiumServerHost: SettingKey[String]   = settingKey[String]("Url of the compendium server")
+    lazy val compendiumServerPort: SettingKey[Int]      = settingKey[Int]("Port of the compendium server")
+    lazy val compendiumFormatSchema: SettingKey[String] = settingKey[String]("Schema type to download")
```

Comment:
Let's follow the same pattern as in https://github.com/higherkindness/mu-scala/blob/master/modules/srcgen/plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala#L36